docs/docs/development/hardware-integration/new-shield.mdx

```diff
@@ -197,190 +281,152 @@ this might look something like:
-        , <&pro_micro 10 GPIO_ACTIVE_HIGH> // col6 in the schematic
+        = <&pro_micro 19 GPIO_ACTIVE_HIGH>
+        , <&pro_micro 18 GPIO_ACTIVE_HIGH>
+        , <&pro_micro 15 GPIO_ACTIVE_HIGH>
         ;
 };
 ```
 
-```dts title=iris_right.overlay
-#include "iris.dtsi"
-
-&default_transform { // The matrix transform for this board is 6 columns over because the left half is 6 columns wide according to the matrix.
-    col-offset = <6>;
-};
+```dts title="my_keyboard_right.overlay"
+#include "my_keyboard.dtsi" // The main dtsi files are included in the overlay.
 
+// Label of the kscan node in the dtsi
 &kscan0 {
     col-gpios
-        = <&pro_micro 10 GPIO_ACTIVE_HIGH> // col6 in the schematic
-        , <&pro_micro 16 GPIO_ACTIVE_HIGH> // col5 in the schematic
-        , <&pro_micro 14 GPIO_ACTIVE_HIGH> // col4 in the schematic
-        , <&pro_micro 15 GPIO_ACTIVE_HIGH> // col3 in the schematic
-        , <&pro_micro 18 GPIO_ACTIVE_HIGH>  // col2 in the schematic
-        , <&pro_micro 19 GPIO_ACTIVE_HIGH>  // col1 in the schematic
+        = <&pro_micro 10 GPIO_ACTIVE_HIGH>
+        , <&pro_micro 11 GPIO_ACTIVE_HIGH>
+        , <&pro_micro 13 GPIO_ACTIVE_HIGH>
         ;
 };
-
 ```
 
-See the [Keyboard Scan configuration documentation](../../config/kscan.md) for details on configuring the KSCAN driver.
-
-### .conf files (Split Shields)
-
-While unibody boards only have one .conf file that applies configuration characteristics to the entire keyboard,
-split keyboards are unique in that they contain multiple .conf files with different scopes.
-For example, a split board called `my_awesome_split_board` would have the following files:
-
-- `my_awesome_split_board.conf` - Configuration elements affect both halves
-- `my_awesome_split_board_left.conf` - Configuration elements only affect left half
-- `my_awesome_split_board_right.conf` - Configuration elements only affect right half
+</TabItem>
+</SplitTabs>
 
-In most case you'll only need to use the .conf file that affects both halves of a split board. It's used for adding features like deep-sleep or rotary encoders.
+### Matrix Transform
 
-```ini title=my_awesome_split_board.conf
-CONFIG_ZMK_SLEEP=y
-```
+The matrix transform is used to transform row/column events into "key position" events.
 
-:::note
-The shared configuration in `my_awesome_split_board.conf` is only applied when you are building with a [`zmk-config` folder](../local-toolchain/build-flash.mdx#building-from-zmk-config-folder) and when it is present at `config/my_awesome_split_board.conf`. If you are not using a `zmk-config` folder, you will need to include the shared configuration in both `my_awesome_split_board_left.conf` and `my_awesome_split_board_right.conf` files.
-:::
-
-</TabItem>
-</Tabs>
+When a key is pressed, a kscan event is generated from it with a row value and a column value corresponding to the index (from 0) of the `row-gpios` pin and `col-gpios` pin respectively that triggered the event.
```

Comment:
```suggestion
When a key is pressed, a kscan event is generated from it with a `row` and a `column` value corresponding to the zero-based indices of the `row-gpios` and `col-gpios` pins that triggered the event, respectively.
```
I think the sentence didn't flow well; this seems an improvement to me although it isn't perfect.